task/06-conditions-n-loops-tasks.js

```diff
@@ -82,7 +97,10 @@ function getSumBetweenNumbers(n1, n2) {
  *   10,10,10 =>  true
  */
 function isTriangle(a,b,c) {
-    throw new Error('Not implemented');
+    if ( a + b > c && a + c > b && b + c > a){
+        return true;
+    } 
+    return false
```

Comment:
You can return it in 1 row, like:
return  a + b > c && a + c > b && b + c > a;
Expression will be truthfully or falsy :)